sympy/physics/units/__init__.py

```diff
@@ -230,39 +230,53 @@ def find_unit(quantity, unit_system="SI"):
 
     >>> from sympy.physics import units as u
     >>> u.find_unit('charge')
-    ['C', 'coulomb', 'coulombs', 'planck_charge', 'elementary_charge']
+    ['C', 'charge', 'coulomb', 'coulombs', 'planck_charge', 'elementary_charge']
     >>> u.find_unit(u.charge)
     ['C', 'coulomb', 'coulombs', 'planck_charge', 'elementary_charge']
```

Comment:
Why can't this find 'charge'?

Reply:
u.find('charge') for ` unit as string`
u.find(u.charge)  for `unit as unit`